src/ontology/edits/oeo-physical.omn

```diff
@@ -1486,15 +1486,18 @@ Class: OEO_00000173
     Annotations: 
         <http://purl.obolibrary.org/obo/IAO_0000115> "A fuel is a portion of matter that has the disposition to be an energy carrier and which has a fuel role that is realised in processes that release the carried energy by transforming the portion of matter into a different kind of portion of matter in a way that releases heat or does work.",
         <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/227
-pull request: https://github.com/OpenEnergyPlatform/ontology/pull/279",
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/279
+add role fuel commodity
+https://github.com/OpenEnergyPlatform/ontology/pull/748",
         rdfs:label "fuel"
     
     EquivalentTo: 
         (<http://purl.obolibrary.org/obo/RO_0000087> some OEO_00000001)
          and (<http://purl.obolibrary.org/obo/RO_0000091> some OEO_00000151)
     
     SubClassOf: 
-        OEO_00000331
+        OEO_00000331,
+        <http://purl.obolibrary.org/obo/RO_0000087> some OEO_00020066
```

Comment:
This says that every `fuel` has a `fuel commodity role`. Shouldn't it be the other way round? Every `fuel commodity` should be a `fuel`.

Reply:
I just realized, why I implemented it this way: I can't use implement `fuel commodity` as (`has role some fuel role`) in oeo-shared because `fuel role` is part of oeo-physical. 
But I can use `fuel commodity role` from oeo-shared in oeo-physical.

Reply:
That's a good point. Can you add the axiom `fuel commodity has role some fuel role` in oeo-physical? This way we would split the class `fuel commodity` and its axioms into different files, but it should solve the problem.

Reply:
If we leave it like that, I think we can delete the equivalent class 'fuel commodity' and just use it as anonymous class. Any other proposals?